package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "orleans.streams.kafka",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
can you bump the minor, since it's a new feature? Thanks.

Reply:
Bumped up the minor version and updated the changelog. @jonathansant